hooks/hook-functions.py

```diff
@@ -0,0 +1,3 @@
+from PyInstaller.utils.hooks import copy_metadata, get_package_paths
```

Comment:
please add a comment what this file does / why we need this (it will not be obvious to readers in the future that this was caused by JIT requirements)

also, unused imports